C-18-01-Capstone.Main/DataContext/Country.cs

```diff
@@ -13,7 +14,270 @@ public class Country
         public string CountryIsoCode2 { get; set; }
 
         public string Name { get; set; }
-        
+
         public ICollection<User> Users { get; set; }
+        
+        public void AddAllCountriesToDB()
```

Comment:
Let's move this to Seed method of DbInitializer. Keep list of companies is not a responsibility of DTO